nengo/tests/test_config.py

```diff
@@ -49,18 +49,18 @@ def test_config_basic():
     assert model.config[a].something is None
 
     with pytest.raises(AttributeError):
-        model.config[a].something_else
+        assert model.config[a].something_else
     with pytest.raises(AttributeError):
-        model.config[a2b].something
+        assert model.config[a2b].something
     with pytest.raises(AttributeError):
         model.config[a].something_else = 1
     with pytest.raises(AttributeError):
         model.config[a2b].something = 1
 
     with pytest.raises(ConfigError, match="'str' is not set up for configuration"):
-        model.config["a"].something
+        assert model.config["a"].something
     with pytest.raises(ConfigError, match="'NoneType' is not set up for configuration"):
-        model.config[None].something
+        assert model.config[None].something
```

Comment:
I know why `pylint` wants this, but in this case, I feel like it's misleading because it looks like the statement is what we're asserting, rather than the error.

Could we do `_ = model.config[None].something`? Or would `pylint` complain about that, too.

Reply:
I've been changing these to `print`s instead of asserts. It silences pylint, which is good, and if the statement is raising an error (as it should) it won't print anything. It has the side benefit that if it doesn't raise an error then it prints out the thing that you're not supposed to be able to do.